lib/ignore/ignore.go

```diff
@@ -406,12 +406,20 @@ func loadParseIncludeFile(filesystem fs.Filesystem, file string, cd ChangeDetect
 	}
 
 	if cd.Seen(filesystem, file) {
-		return nil, parseError(fmt.Errorf("multiple include of ignore file %q", file))
+		return nil, fmt.Errorf("multiple include of ignore file %q", file)
```

Comment:
```suggestion
		return nil, errors.New("multiple include")
```

(looks like the calling function adds the filename as context already)